cmd/relayer/main.go

```diff
@@ -28,10 +28,8 @@ const (
 	flagForwarderAddress = "forwarder-address"
 	flagKey              = "key"
 	flagRPCPort          = "rpc-port"
-	flagDev              = "dev"
-	flagLog              = "log"
-
-	defaultGanacheKey = "4f3edf983ac636a65a842ce7c78d9aa706d3b113bce9c46f30d7d21715b23b1d"
```

Comment:
That's the Alice/XMRTaker key in atomic swap. I figured it was safest to just get rid of it so the user actually knows which key is being used.